docs/DeveloperGuide.md

```diff
@@ -2,7 +2,8 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
+### Table of Contents
```

Comment:
Not sure about making this a heading, but I thought it made more sense than leacing it as a bullet point. Wdyt?

Reply:
Nice!